router/throttler/factory_test.go

```diff
@@ -71,6 +74,27 @@ func TestFactory(t *testing.T) {
 			return ta.getLimit() == int64(0)
 		}, 2*time.Second, 100*time.Millisecond)
 	})
+
+	t.Run("check stats when adaptive is enabled", func(t *testing.T) {
+		config := config.New()
```

Comment:
`Variable 'config' collides with imported package name`. let's some other variable name in here, pls?
```suggestion
		conf := config.New()
```

Reply:
Done